src/interfaces/ecpg/ecpglib/misc.c

```diff
@@ -303,7 +303,7 @@ ecpg_log(const char *format,...)
 	/* dump out internal sqlca variables */
 	if (ecpg_internal_regression_mode && sqlca != NULL)
 	{
-		fprintf(debugstream, "[NO_PID]: sqlca: code: %ld, state: %s\n",
+		fprintf(debugstream, "[NO_PID]: sqlca: code: %ld, state: %.5s\n",
```

Comment:
Someday the sqlstate field size may be changed. What about
```C
fprintf(debugstream, "[NO_PID]: sqlca: code: %ld, state: %.*s\n",
        sqlca->sqlcode, sizeof(sqlca->sqlstate), sqlca->sqlstate);
```
?

Reply:
> Someday the sqlstate field size may be changed

it is impossible



> What about ... ?

applied